### PR TITLE
[wip] chore: add release wg codeownership of script

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

Update primary codeownership of the `script/` folder to `@wg-releases`. This is a broad stroke, as not every single file in there is release-specific.  However, I would hedge that 95% of changes to that folder are in fact release-specific, so this seems to me to be more efficient than marking specific files under the new WG.

cc @electron/wg-releases 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -18,4 +18,7 @@
 /default_app/ @electron/docs
 /docs/ @electron/docs
 /docs-translations/ @electron/i18n
-/npm/ @electron/electrocats
\ No newline at end of file
+/npm/ @electron/electrocats
+
+# WG CODEOWNER Patterns
+/script/ @electron/wg-releases
```
Comment (nornagon) [inline .github/CODEOWNERS]: this seems overly broad. Can we move all release-related scripts to e.g. `script/release` and protect that instead?

Comment (codebytere) [reply .github/CODEOWNERS]: my concern is that it'd break all our backports and i'd love to not do that i think

Comment (codebytere) [reply .github/CODEOWNERS]: i can try though :)

**Comment by nornagon** [comment]
well, i'm 👎 on this idea at all then.

**Comment by codebytere** [comment]
Please hold on merging this until i've had time to think it through a little more, since it would cascade into changes with sudowoodo I'd potentially like to avoid.

**Comment by codebytere** [comment]
This needs some work and may be best done in a slightly different way in the near future.

**Pull request closed without merging**